main.js

```diff
@@ -1,12 +1,49 @@
 'use strict';
 
-const logger = require('ft-next-logger');
+const logger = require('ft-next-logger').logger;
+const fetchres = require('fetchres');
+const raven = require('raven');
+let ravenMiddleware;
+
+function sendErrorDev (err, req, res) {
+	if (err.name === fetchres.ReadTimeoutError.name) {
+		logger.error('Dependency timeout -', err);
```

Comment:
Neat. But feels like that sort of gymnastics is worth building into next logger, which I could do later today if it seems like a good idea